runtime/isp_kernel.py

```diff
@@ -117,15 +143,26 @@ def main():
     parser.add_argument("-d", "--debug", action="store_true", help='''
     Enable debug logging
     ''')
+    parser.add_argument("-s", "--source", type=lambda s: s.split(','), help='''
```

Comment:
I don't see the need to make this a list of dirs, since this tool only installs one policy at a time

Reply:
I don’t have all my DPL source in one directory. 

For example, think back to the web app ppac policy when we “hooked” the malloc policy. We should be able to put our source DPL in the project directory and say `import osv.frtos.main.heap` and then the tool  goes and finds that DPL for me. 

This mirrors programming in C. I want to be able to call malloc without hunting down the libc source and putting it into my project directory